assets/size-guide.js

```diff
@@ -0,0 +1,97 @@
+// Constants
+const sizeGuideModal = document.querySelector('[data-size-guide-modal]')
+const sizeGuideModalOpenButton = document.querySelector('[data-size-guide-modal-open-button]')
+const sizeGuideModalCloseButton = document.querySelector('[data-size-guide-modal-close-button]')
+const overlayForOpenSizeGuideModal = document.querySelector('[data-size-guide-modal-overlay]')
+const focusableElements = sizeGuideModal.querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])')
+const focusableElementsArray = [...focusableElements]
+
+// Utility Functions
+const addClass = string => el => el.classList.add(string)
+const removeClass = string => el => el.classList.remove(string)
+const hide = addClass('hidden')
+const makeBlue = addClass()
+const show = removeClass('hidden')
+const map = fn => arr => arr.map(fn)
+const showAll = map(show)
+const hideAll = map(hide)
+const contains = string => el => el.classList.contains(string)
+const isHidden = contains('hidden')
+const bodyMakeOverflowHidden = () => {
+  document.body.style.overflow = 'hidden'
+}
+const bodyRemoveOverflowStyle = () => {
+  document.body.style.removeProperty('overflow')
+}
+const modulo = (x,y) => ((y % x) + x)  % x
+const combine = (a, b) => a + b
+
+// Functions
+const keyPressIs = (nameOfKey, event) => event.code === nameOfKey 
+
+const openSizeGuideModal = () => {
+  showAll([sizeGuideModal, overlayForOpenSizeGuideModal])
+  bodyMakeOverflowHidden()
+  listenToKeydownForModal()
+}
+
+const closeSizeGuideModal = () => {
+  hideAll([sizeGuideModal, overlayForOpenSizeGuideModal])
+  bodyRemoveOverflowStyle()
+  removeKeydownEventListenersForModal()
+}
+
+const closeSizeGuideModalOnEsc = (event) => {
+  if ( keyPressIs('Escape', event) && !isHidden(sizeGuideModal) ) closeSizeGuideModal()
+}
+
+const closeSizeGuideModalOnEnter = (event) => {
+  if (keyPressIs('Enter', event) && event.target === sizeGuideModalCloseButton) closeSizeGuideModal()
+}
+
+const listenToKeydownForModal = () => {
+  document.addEventListener('keydown', closeSizeGuideModalOnEsc)
+  document.addEventListener('keydown', closeSizeGuideModalOnEnter)
+  sizeGuideModal.addEventListener('keydown', trapFocusInModal)
+}
+
+const removeKeydownEventListenersForModal = () => {
+  document.removeEventListener('keydown', closeSizeGuideModalOnEsc)
+  document.removeEventListener('keydown', closeSizeGuideModalOnEnter)
+  sizeGuideModal.removeEventListener('keydown', trapFocusInModal)
+}
+
+const getTabDirection = event => {
+  if (keyPressIs('Tab', event)) {
+    if (event.shiftKey) {
+      return -1
+    } else return 1
```

Comment:
Convention is to write like this

```
    if (event.shiftKey) {
      return -1
    } 
    return 1
```
The reason is because if the if statement is true, you return -1 and never get to returning 1, so don't have to worry about doing an else statement, basically you avoid writting 4 more letters, yay, how efficient.

This could also be a ternary `return event.shiftKey ? -1 : 1`